services/placement/algo/helper.go

```diff
@@ -338,7 +366,12 @@ func newHeap(
 	targetLoad map[string]int,
 	rackToWeightMap map[string]uint32,
 ) *instanceHeap {
-	hHeap := &instanceHeap{capacityAscending: capacityAscending, instances: instances, targetLoad: targetLoad, rackToWeightMap: rackToWeightMap}
+	hHeap := &instanceHeap{
```

Comment:
Perhaps name the local variable here just `h`?  To avoid the strange `hHeap` name?